### PR TITLE
added note for optional steps for IdP configuration

### PR DESCRIPTION
**Pull request by TheFuzzard**
a lot of people that are trying to create an IdP for a workforce scenario are confused by this step as it is not needed for them and the configuration is already complete by this point

## Description:
- **What's changed?** 

add a note on the "Register an app in Okta" step for configuring identity providers that it is not mandatory to continue with next steps in order to use the provider

- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

**Review by yuliu-okta — APPROVED** [review]

**Comment by IanHakes-okta** [comment]
Text changes haven't been made

**Review by IanHakes-okta — CHANGES_REQUESTED** [review]
Please update text note according to comment below

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/register-app-in-okta/index.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ---
 title: Register an App in Okta
 ---
+
+> **Note:** If you only need to authenticate users into Okta and not have them redirected to a particular OIDC client the IdP configuration would be complete at this point and [routing rules](https://help.okta.com/en/prod/Content/Topics/Security/Identity_Provider_Discovery.htm) have to be set to redirect users from the Okta login page to the correct provider.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/register-app-in-okta/index.md:5]: @IanHakes-okta 
If you need Okta only to authenticate users and not to redirect them to a particular OpenID Connect client, then the Identity Provider (IdP) configuration is complete. Add routing rules to redirect users from the Okta Sign-In Page to the IdP.

Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/register-app-in-okta/index.md:5]: @yuliu-okta

**Comment by susanharper-okta** [comment]
this one can be closed

**Pull request closed without merging**